apps/meteor/client/startup/roles.ts

```diff
@@ -32,8 +32,11 @@ Meteor.startup(() => {
 		if (!Meteor.userId()) {
 			return;
 		}
-		sdk.stream('roles', ['roles'], (role) => {
-			events[role.type]?.(role);
+
+		Tracker.afterFlush(() => {
+			sdk.stream('roles', ['roles'], (role) => {
+				events[role.type]?.(role);
+			});
 		});
```

Comment:
could you please check if this solves the issue?
```suggestion
		if (Meteor.status().status !== 'connected') {
			return;
		}
		sdk.stream('roles', ['roles'], (role) => {
			events[role.type]?.(role);
		});
```

Reply:
I checked and don't work :/